app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesController.scala

```diff
@@ -107,6 +108,7 @@ class ExemptionAndLossesController @Inject() (
         _.disposalDate,
         c => Some(c.disposalDate)
       ),
+      _ => None, // no disposalDate for ExampleCompanyDetailsAnswers
```

Comment:
the disposal date is the same as the completion date for shares - you can construct a `DisposalDate` by putting the completion date and the tax year from the multiple disposal triage anwers